src/pages/index.tsx

```diff
@@ -40,7 +40,6 @@ const MainSection = () => (
       alignItems={{ base: 'center', lg: 'start' }}
       spacing="8"
       flexGrow={1}
-      flexBasis={0}
```

Comment:
Just removing this fixes the layout for Safari on iOS 14! Not really sure why I added this property, but it seems like a leftover from a previous layout implementation.